src/XMakeTasks/Exec.cs

```diff
@@ -299,6 +217,13 @@ private void CreateTemporaryBatchFile()
                 sw.WriteLine("set errorlevel=dummy");
                 sw.WriteLine("set errorlevel=");
 
+                // We probably have to change the code page to UTF8 (65001) for non-ansi characters to work.
+                if (EncodingUtilities.CurrentSystemOemEncoding.CodePage != sw.Encoding.CodePage)
+                {
+                    // Output to nul so we don't change output and logs.
+                    sw.WriteLine($"chcp {sw.Encoding.CodePage}>nul");
```

Comment:
Good point, probably best to just not go there at this point. Especially for a single change and I'm not sure what it would do to the xplat / Xamarin Studio story when this gets merged.